src/utilities/report-utils.ts

```diff
@@ -47,18 +48,17 @@ const makeSourceKey = (url: string | null) => {
 export const getReportUrl = () => {
   const reportLink = portalReportBaseUrl();
   const reportFirebaseApp = firebaseAppName();
-  const activity = queryValue("activity");
-  const activityUrl = activity? ((activity.split(".json"))[0]).replace("api/v1/","") : "";
+  const resourceUrl = getResourceUrl();
   const runKey= queryValue("runKey");
   // Sometimes the location of the answers is overridden with a report-source param
   const answerSource = queryValue("report-source") || getCanonicalHostname();
-  const sourceKey = activityUrl ? makeSourceKey(activityUrl) : getCanonicalHostname();
+  const sourceKey = resourceUrl ? makeSourceKey(resourceUrl) : getCanonicalHostname();
 
   if (runKey) {
     return reportLink
             + "?"
             + "runKey=" + runKey
-            + "&activity=" + activityUrl
+            + "&activity=" + resourceUrl
```

Comment:
It seems better if we changed this parameter to be resourceUrl. But because that requires changes to the portal-report codebase to support. I'd suggest changing this to:
```
// send the resourceUrl in the activity parameter for backwards compatibility
+ "&activity=" + resourceUrl
+ "&resourceUrl=" + resourceUrl
```

Then once the portal-report is updated to handle the resourceUrl we can remove this activity parameter here.

Reply:
@scytacki OK, makes sense 👍